labelbox/schema/identifiable.py

```diff
@@ -0,0 +1,65 @@
+from abc import ABC, abstractmethod
+from enum import Enum
+from typing import List, Union
+
+
+class IdType(str, Enum):
+    DataRowId = "UID"
+    GlobalKey = "GLOBAL_KEY"
+
+
+class Identifiable(ABC):
```

Comment:
Confused why this is a wrapper around an array, from initial designs this is basically a wrapper around a single id.

Reply:
Maybe its a matter of renaming it to Identifiables

Reply:
The idea with Identifiable ABC is that its an interface that lets code know the object is an entity with a unique Id and it provides some function for code to fetch the id structure.  This is more like a utility class, but since its a list you should implement __iter__ and other functions so it can act like a collection.

Reply:
Implemented as an iter, that is a good idea thank you @sqlboy